run-zfs-tests

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+set -o pipefail
+
+testvm=$1
+if [ "$testvm" == "" ] ; then
+    >&2 echo "First parameter must be the VM where you will run the tests."
+    >&2 echo "Ideally qubes.VMShell service is 100% permitted from this VM to that VM."
+    exit 2
+fi
+shift
+
+bombshell-client "$testvm" which zfs >/dev/null || {
```

Comment:
I'd want to request using `qvm-run-vm` here (as it is in standard setup, contrary to bombshell-client). But, while `qvm-run` learned proper argument quoting and multiline commands handling, `qvm-run-vm` did not yet. And `qvm-run` (which you _can_ install in a VM) requires quite a bit more permissions (at the very least - listing VMs in the system).
So, for the time being, lets keep it this way... But please add a comment how one can get this tool.

Reply:
What about using `qubes.VMExec` manually?